conf/default-vars.yaml

```diff
@@ -48,7 +48,10 @@ vars:
 
   border:
     size: 1px
-    radius: 3px
+    radius:
+      default: 3px
+      lg: 5px
```

Comment:
I would argue that if you're using border-radius on any element, design-wise you'd want them all to match for consistency.  I'd imagine its more the exception to have something with drastically different border-radius.